### ISSUE TITLE
Questions about calculating the accuracy of unseen classes

### ISSUE THREAD

**Issue by Programmergg** [zcrwind/tgg-pytorch#4]
![微信图片_20201005153544](https://user-images.githubusercontent.com/32862873/95052083-e3600200-0720-11eb-8dd2-6459a9d8d8c0.png)
I have a big question. Why do you use the training set aggregation method when calculating the accuracy of unseen classes? Did you forget to change the code when you published it, or did you write it like this?

**Comment by shiming-chen** [comment]
I also see such problem. I try to correct it with the unseen set aggregation, but there are bugs. Have you solve it?   @Programmergg @zcrwind
